### DISCUSSION TITLE
Student pack rejected since program is online

### DISCUSSION THREAD

**Discussion by andreialbu03** [community/community#54167] [GitHub Education]
### Hi everyone,

Hi,

I am unable to verify for the GitHub education student pack despite being a post-secondary student. I am a student at an online university located in one part of my country while I am located elsewhere (different provinces). I believe this difference in location is why I cannot get automatically authenticated by the system.

I contacted support but they closed the ticket and said I should post it in this forum... Anyone know how to get verified?

**Comment by samus-aran** [comment]
Hi @andreialbu03 

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Comment by andreialbu03** [comment]
Hello @samus-aran,

Thanks for your response. The school is located across the country so it is impossible for me to apply while on campus. Additionally, the documents I have do not say the program is remote, they simply indicate the name of the program (Bachelor of Computer Science). 

I have attached the response I received from support, it simply includes various links.

Is there no way that a support agent can look over my file and manually approve it? There are certain extenuating circumstances that the automatic system does not work for, such as my case.

<img width="955" alt="Screenshot 2023-05-02 at 09 29 55" src="https://user-images.githubusercontent.com/74505056/235680953-72c09278-8ae7-465d-86c5-7e0950945131.png">
